docker-compose-image-tag.yml

```diff
@@ -33,7 +33,11 @@ services:
       - redis:/data
 
   db:
-    env_file: docker/.env
+    env_file:
```

Comment:
these changes for env_file seems make i can not start superset following the documentation by
`docker-compose -f docker-compose-image-tag.yml up`

the error:
```The Compose file './docker-compose-image-tag.yml' is invalid because:
services.db.env_file contains {"path": "docker/.env", "required": true}, which is an invalid type, it should be a string
services.superset.env_file contains {"path": "docker/.env", "required": true}, which is an invalid type, it should be a string
services.superset-init.env_file contains {"path": "docker/.env", "required": true}, which is an invalid type, it should be a string
services.superset-worker.env_file contains {"path": "docker/.env", "required": true}, which is an invalid type, it should be a string
services.superset-worker-beat.env_file contains {"path": "docker/.env", "required": true}, which is an invalid type, it should be a string```

I am on Ubuntu 22.04.4 LTS

Reply:
hey @shaoxp , this got fixed in https://github.com/apache/superset/pull/28039 , but note that you need docker-compose 2.24+ for this config to work. I tried to lookup if there's a version of the yaml I can setup to inform users to upgrade but couldn't find anything.

Reply:
Thanks for the info. i am able to start locally. 

maybe you can update the document to include the required version.